invenio_app_ils/ui/backoffice/src/common/api.js

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
+
+// In development create a personal token to be able to perform requests and
+// and put it in a .env file under the name `REACT_APP_JWT_TOKEN`
+const token =
+  process.env.NODE_ENV === 'development'
+    ? process.env.REACT_APP_JWT_TOKEN
+    : document.getElementsByName('authorized_token')[0].value;
+
+const axiosIls = axios.create({
```

Comment:
I would rename that in $axios or $http. Also what about exporting in case we need it an other file?

Reply:
I was looking for a name, prefixing it with `$` didn't even cross my mind, cheers! Will export it too!